src/main/java/com/google/sps/data/GovAdaptor.java

```diff
@@ -0,0 +1,71 @@
+/** A class, classified as an Adaptor. this is the Adaptor class for the government data source
+ * the adaptors get their information from their respective data srouces - the government API is called here. 
+*/
```

Comment:
This should be moved to just above the class declaration.

Typo: srouces instead of sources